packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.css

```diff
@@ -1,7 +1,3 @@
-:host {
-  --slds-c-toast-sizing-min-width: 100%;
```

Comment:
Did we need this anywhere?

Reply:
nope, mehdi raised the same comment, I tested it and it seem it was not needed